.coveragerc

```
@@ -7,3 +7,6 @@ disable_warnings =
 
 [report]
 show_missing = True
+exclude_lines = 
+    pragma: no cover
+    if TYPE_CHECKING:
```

Alternatively there's https://coverage.readthedocs.io/en/latest/config.html#report-exclude-also , depending on how you handle your skeleton configs.

```suggestion
exclude_also = 
    if TYPE_CHECKING:
```

---

After re-reading, the doc, this seems to be a more modern/preferred way of doing it.